data_pipeline/dataset.py

```diff
@@ -22,16 +23,19 @@
 SENTENCE_KEY = 'sentence'
 SENTENCE_LEN_KEY = 'sentence_lengts'
 CONCEPTS_KEY = 'concepts'
+GLOVE_CONCEPTS_KEY = 'glove_concepts'
 CONCEPTS_LEN_KEY = 'concepts_lengths'
 ADJ_MAT_KEY = 'adj_mat'
 AMR_STR_KEY = 'amr_str'
+CONCEPTS_STR_KEY = 'concepts_str_key'
```

Comment:
Why not 'concepts_strings' instead of 'concepts_str_key'? I was thinking this will keep the naming conventions used for the other.

Reply:
Yes you are right here, it slipped. Renamed!